apps/client/src/app/pages/portfolio/x-ray/x-ray.component.html

```diff
@@ -0,0 +1,129 @@
+<div class="container mt-5">
+  <div class="row">
+    <div class="col">
+      <h2 class="h3 mb-3 text-center">X-ray</h2>
+      <p class="mb-4">
+        <span i18n
+          >Ghostfolio X-ray uses static analysis to identify potential issues
+          and risks in your portfolio.</span
+        >
+        <span class="d-none"
+          >It will be highly configurable in the future: activate / deactivate
```

Comment:
`Activate / deactivate rules and customize the thresholds to match your personal investment style.`